erp/export/create_schema.py

```diff
@@ -0,0 +1,93 @@
+from frictionless import Schema, Field
+
+from erp import schema
+from erp.export.models import ETALAB_SCHEMA_FIELDS
+from erp.schema import FIELDS
+
+
+def generate_schema(base="base-schema.json", outfile="schema.json"):
+    table_schema = Schema(base)
+    # table_schema["version"] = "0.0.1"
```

Comment:
This raises a good question; how and where should we manage the schema version? Ideally in a single place…